### PR TITLE
3.0 - Prefix elements paths

### PR DESCRIPTION
**Pull request by HavokInspiration**
Follow changes made to cakephp/cakephp#6233

### CODE REVIEW

File: en/views.rst [hunk 1]
```diff
@@ -637,6 +637,20 @@ For elements inside subfolder of a plugin
     echo $this->element('Contacts.sidebar/helpbox');
 
 
+Routing prefix and Elements
+---------------------------------
+
+.. versionadded:: 3.1
```
Comment (markstory) [inline en/views.rst]: This is 3.0.1 now isn't it?

File: en/views.rst [hunk 2]
```diff
@@ -637,6 +637,20 @@ For elements inside subfolder of a plugin
...
+
+If you have a Routing prefix configured, the Element path resolution can switch to
+a prefix location, as Layouts and action View do.
+Assuming you have a prefix "Admin" configured and you call::
+
+ echo $this->element('my_element');
+
+The element first be looked for in ``src/Template/Admin/Element/``. If such a file
```
Comment (antograssiot) [inline en/views.rst]: path are now bold as per guidelines, same bellow

File: en/views.rst [hunk 3]
```diff
@@ -637,6 +637,20 @@ For elements inside subfolder of a plugin
...
+Routing prefix and Elements
+---------------------------------
+
+.. versionadded:: 3.1
+
+If you have a Routing prefix configured, the Element path resolution can switch to
+a prefix location, as Layouts and action View do.
+Assuming you have a prefix "Admin" configured and you call::
+
+ echo $this->element('my_element');
```
Comment (antograssiot) [inline en/views.rst]: doesn't this need indentation to shows up properly ?

**Comment by HavokInspiration** [comment]
@markstory @antograssiot I will make the changes tonight.

**Comment by HavokInspiration** [comment]
Done

**Comment by markstory** [comment]
Thanks :+1:

**Pull request merged**